data.sql

```diff
@@ -5,4 +5,14 @@ VALUES
 ('Agumon', '2020-02-03', 0, true, 10.23),
 ('Gabumon', '2018-11-15', 2, true, 8),
 ('Pikachu', '2021-01-07', 1, false, 15.04),
-('Devimon', '2017-05-12', 5, true, 11);
\ No newline at end of file
+('Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO animals (name, date_of_birth, weight, neutered, escape_attempts)
+VALUES
+('Charmander', '2020-02-08', -11, false, 0),
+('Plantmon', '2021-11-15', -5.7, true, 2),
+('Squirtle', '1993-04-02', -12.13, false, 3),
+('Angemon', '2005-01-12', -45, true, 1),
+('Boarmon', '2005-01-07', 20.4, true, 7),
```

Comment:
- [ ] The data_of_birth is wrong here, kindly make sure to correct these values the month in these two entries is `Jun` so the data should be `2005/06/12` and `2005/06/07`

Reply:
👍